pkg/cmd/cli/datamover/backup.go

```diff
@@ -127,7 +128,8 @@ func newdataMoverBackup(logger logrus.FieldLogger, factory client.Factory, confi
 		return nil, errors.Wrap(err, "error to create client config")
 	}
 
-	ctrl.SetLogger(zap.New(zap.UseDevMode(true)))
```

Comment:
FYI zap is being removed in this PR.

It was meant to be removed here https://github.com/vmware-tanzu/velero/pull/2561/files#r432107463 and was kept here https://github.com/vmware-tanzu/velero/pull/8046#discussion_r1696542433

This means all logs are going to be affected by the default logger flags, that means if user want all logs in json, they will get all logs in json.